packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -150,7 +153,7 @@ export const GridCell: React.FC<GridCellProps> = React.memo((props) => {
       aria-colindex={colIndex}
       style={style}
       /* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */
-      tabIndex={hasFocus ? 0 : -1}
+      tabIndex={isCellFocusable ? 0 : -1}
```

Comment:
it's not because the cell is focusable that it has the focus. 
If it has the focus then tabIndex = 0, not when it's focusable.
Some refactoring to do here...

Reply:
Yes, that is what `isCellFocusable` do, probably better wording would be `isCellFocused`?

Reply:
> If it has the focus then tabIndex = 0, not when it's focusable.

@dtassone This is true, but with one important limitation. The cell having the focus is not the only case when we set a cell tabbable (`tabIndex={0}`). It's also occurs for the initial rendering, so the first cell can be <kbd>Tab</kbd> on.